var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -91,6 +95,14 @@ def configure_args(self):
         if '+perl' in spec:
             args.append('PERL={0}'.format(spec['perl'].command.path))
 
+        if '+nls' in spec:
+            args.extend([
```

Comment:
Can you also explicitly `--enable-nls` in this branch? Explicit is better than implicit and works even if the default is ever changed or if the default changes based on whether or not libs are found.

Reply:
Oh, sure! Only `--disable-nls` is shown in `./configure --help` and I'd been playing with `ffmpeg` recently which has a handmade configure script that doesn't actually allow `--enable-*` for every `--disable-*` option available. It looks like that totally works for real autoconf, though.